src/components/Input.tsx

```diff
@@ -7,12 +7,11 @@ interface InputProps {
 
 function Input({label, placeholder, value, onInput}: InputProps) {
   return (
-    <div style={{width: '100%'}}>
+    <div className="w-[100%]">
```

Comment:
bootstrap way https://getbootstrap.com/docs/5.0/utilities/sizing/#relative-to-the-parent